atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -157,13 +162,6 @@ static boolean uniqueSchemaWithQuorumAgreementAndOtherNodesUnreachable(
                 .collect(Collectors.toList());
     }
 
-    private static int getNumberOfReachableNodes(Map<String, List<String>> versions) {
-        return versions.entrySet().stream().filter(entry -> !entry.getKey().equals(VERSION_UNREACHABLE))
-                .map(Entry::getValue)
-                .mapToInt(List::size)
-                .sum();
-    }
-
     private static long sleepAndGetNextBackoffTime(long sleepTime) {
         try {
             Thread.sleep(sleepTime);
```

Comment:
Maybe could use some logging here? We don't expect any to happen if there's only a single mutation happening at a time, but when there are many, it would probably help with figuring out what's going on.

Reply:
Added an info log line, it doesn't provide details, but that's probably fine as we get details if we fail to converge on the schema version